popup.js

```diff
@@ -203,4 +203,73 @@ function loadpages() {
 }
 
 
+//form validation
+const form=document.getElementById("form");
+const names=document.getElementById("names");
+const email=document.getElementById("email");
+const coment=document.getElementById("coment");
+
+form.addEventListener("submit", e =>{
+      e.preventDefault();
+    validateinputs();    
+})
+
+const isValidEmail = email => {
+    const re = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+    return re.test(String(email).toLowerCase());
+}
+
+const validateinputs=() =>{  
+    const nameinput= names.value.trim();
+    const emailinput= email.value.trim();
+    const comentinput= coment.value.trim();   
+    if(nameinput===""){  const errornames = document.querySelector(".your-name-container");
+    names.placeholder="Please enter your name";
+    errornames.classList.add("error");
+       }
+    else { 
+   const succesnames = document.querySelector(".your-name-container");
+   succesnames.classList.add("succes");
```

Comment:
- [ ] As requested by the previous reviewer, You haven't implemented the error message for the email address if a user enters an email with `uppercase`. Also, the form is not submitted to form spree when you have all your form filled with details

Your Form 👇 

![col](https://user-images.githubusercontent.com/78634841/211172695-07accd53-205d-4bb8-a5b3-f2bde618a54a.png)